src/NLog.StructuredLogging.Json.Tests/LoggerExtensionsScopeTests.cs

```diff
@@ -54,6 +54,21 @@ public void TearDown()
         }
 
         [Test]
+        public void Should_Fail_If_Logger_Is_Null()
+        {
+            ILogger nullLogger = null;
+            var exeption = Assert.Throws<ArgumentNullException>(() => nullLogger.BeginScope("should fail"));
```

Comment:
I think `Assert.Throws` has an overload where you can pass it the expected parameter name as the first argument.